tests/epyccel/test_epyccel_pyc_math.py

```diff
@@ -0,0 +1,132 @@
+
+    assert(f(x) == call_factorial(x))
+# -----------------------------------------------------------------------------
+# New in version 3.9.
+@pytest.mark.skipif(sys.version_info < (3, 9), reason="requires python3.9 or higher")
+@pytest.mark.parametrize( 'language', (
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = [
+            pytest.mark.xfail(reason="lcm not implemented in fortran"),
+            pytest.mark.fortran]
+        )
+    )
+)
+def test_call_lcm(language):
+    @types(int, int)
+    def call_lcm(x, y):
+        from math import lcm
+        return lcm(x, y)
+
+    f = epyccel(call_lcm, language=language)
+    x = randint(1e9)
+    y = randint(1e9)
+
+    assert(f(x,y) == call_lcm(x, y))
+# -----------------------------------------------------------------------------
+@pytest.mark.parametrize( 'language', (
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = [
+            pytest.mark.xfail(reason="radians not implemented in fortran"),
+            pytest.mark.fortran]
+        )
+    )
+)
+def test_call_radians(language):
+    @types('real')
+    def call_radians(x):
+        from math import radians
+        return radians(x)
+
+    f = epyccel(call_radians, language=language)
+    x = uniform(low=0.0, high=1e6)
+
+    assert isclose(f(x), call_radians(x), rtol=1e-14, atol=1e-14)
+    assert isclose(f(-x), call_radians(-x), rtol=1e-14, atol=1e-14)
+# -----------------------------------------------------------------------------
+@pytest.mark.parametrize( 'language', (
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = [
+            pytest.mark.xfail(reason="degrees not implemented in fortran"),
+            pytest.mark.fortran]
+        )
+    )
+)
+def test_call_degrees(language):
+    @types('real')
+    def call_degrees(x):
+        from math import degrees
+        return degrees(x)
+
+    f = epyccel(call_degrees, language=language)
+    x = uniform(low=0.0, high=1e6)
+
+    assert isclose(f(x), call_degrees(x), rtol=1e-14, atol=1e-14)
+    assert isclose(f(-x), call_degrees(-x), rtol=1e-14, atol=1e-14)
+# -----------------------------------------------------------------------------
+@pytest.mark.parametrize( 'language', (
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = [
+            pytest.mark.xfail(reason="degrees not implemented in fortran"),
+            pytest.mark.fortran]
+        )
+    )
+)
+def test_call_degrees_i(language):
+    @types('real')
+    def call_degrees_i(x):
+        from math import degrees
+        return degrees(x)
+
+    f = epyccel(call_degrees_i, language=language)
+    x = randint(1e6)
+
+    assert isclose(f(x), call_degrees_i(x), rtol=1e-14, atol=1e-14)
+    assert isclose(f(-x), call_degrees_i(-x), rtol=1e-14, atol=1e-14)
+# -----------------------------------------------------------------------------
```

Comment:
Why did you decide to create a new test file rather than putting these tests which test parts of the `math` library with all the other tests for the math library?

Reply:
I know it's the same topic (math), but here we are dealing with our implementation, the kind of errors we may get are different.
I think moving the file to recognized_function dir will be enough.
what do you think?

Reply:
Sounds like a good compromise :+1: